### ISSUE TITLE
Handing SIGPIPE

### ISSUE THREAD

**Issue by 2011eric** [NTU-SP/SP-Hw1-release#45]
I found that when the client disconnects from the server while the server is calling write(), the process of the server will receive signal 'SIGPIPE' and be terminated(Maybe by the OS,I'm not sure).
Should we consider this case?

**Comment by imingasco** [comment]
No. Signal has not been covered yet, so you may assume that client won't disconnect during server is writing.
